### PR TITLE
Bhavita/settings pullfromdb

### PR DESCRIPTION
**Pull request by bhavitaap**
[//]: # "Feel free to customize this template to your project's vibes"

### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- header with name, email, and sign out button 
- settings page information updated to pull from database for all sections except performance interest-group size and notifications

### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

https://github.com/user-attachments/assets/6c823391-049b-44b8-bfbf-20b10ffedc05

## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- fetchVolunteerPreferences and fetchVolunteerInfo in api/supabase/queries/volunteers.ts
- components/SettingsCard <- updated to take in info from current session
- settings/page.tsx

## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- group size and notifications info is still hardcoded bc couldn't find in the database

### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @jxmoose

### CODE REVIEW

File: package.json
Comment (jxmoose) [inline package.json:1]: remove this change

File: app/settings/styles.ts [hunk 1]
```diff
@@ -25,3 +28,37 @@ export const SettingDiv = styled.div`
...
+export const Email = styled(P)`
+  color: ${COLORS.gray10};
+  font-style: normal;
+  font-weight: 400;
+  margin-top: 0.25rem;
+`;
+
+export const SignOutButton = styled.button`
+  border: 1px solid ${COLORS.pomegranate12};
+  padding: 0.5rem 1rem;
```
Comment (jxmoose) [inline app/settings/styles.ts:46]: the second padding should be .75 rem

File: components/SettingsCard/SettingsCardPerformanceInterest.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <styles.Edit src={Edit} alt="Edit" />
+      </styles.AvailabilityHeader>
+      <styles.Content>
+        <div>
+          <styles.SubHeader>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Type of Act
+              </P>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardPerformanceInterest.tsx:25]: use a styled component for this

File: components/SettingsCard/SettingsCardPerformanceInterest.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                    </li>
+                  );
+                })}
+              </ul>
+            </styles.SettingDetail>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Genre
+              </P>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardPerformanceInterest.tsx:45]: component

File: components/SettingsCard/SettingsCardShowPreferences.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        <styles.Edit src={Edit} alt="Edit" />
+      </styles.AvailabilityHeader>
+      <styles.Content>
+        <div>
+          <styles.SubHeader>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Facility Type
+              </P>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardShowPreferences.tsx:25]: component

File: components/SettingsCard/SettingsCardShowPreferences.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                    </li>
+                  );
+                })}
+              </ul>
+            </styles.SettingDetail>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Location Preferences
+              </P>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardShowPreferences.tsx:45]: component

File: components/SettingsCard/SettingsCardShowPreferences.tsx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+                    </li>
+                  );
+                })}
+              </ul>
+            </styles.SettingDetail>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Preferred Audience
+              </P>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardShowPreferences.tsx:65]: disc
